tasks/test-coverage.js

```diff
@@ -87,9 +73,9 @@ var setupBackupAndInstrumentationDir = function() {
   }
 
   log('• copy src files to backup folder');
-  wrench.copyDirSyncRecursive(dir, backupDir, copyOpts);
+  fse.copySync(dir, backupDir);
```

Comment:
The backupDir should be empty at this time, but I will add the options to have the same as we had with wrench.